Makefile

```
@@ -107,7 +107,7 @@ log-check:
 	! grep -n "!! Input index error" hott-online.ilg /dev/null
 
 version.tex:
-	/bin/echo -n '\newcommand{\OPTversion}{' > version.tex
+	/bin/echo -n '\\newcommand{\\OPTversion}{' > version.tex
```

I've run into this issue in my own projects recently on Windows, and the solution I've used is
```suggestion
	printf '\\newcommand{\\OPTversion}{' > version.tex
```